packages/react-query/rollup.config-1739405461180.mjs

```
@@ -0,0 +1,209 @@
+    const runnerRoot = '../..';
+    const analysisFilePath = 'dist/bundle-analysis.json';
+    const previousAnalysisDir = 'downloads/previous-bundle-analysis';
+    const currentPath = path.resolve(packageDir, analysisFilePath);
+    const relativePath = path.relative(path.resolve(runnerRoot, 'packages'), packageDir);
+    const prevPath = path.resolve(runnerRoot, previousAnalysisDir, relativePath, analysisFilePath);
+    return {
+        currentPath,
+        prevPath
+    };
+}
+const ansiRegex = new RegExp('([\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)|(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-ntqry=><~])))', 'g');
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)